src/getProjectId.js

```diff
@@ -21,10 +22,19 @@ module.exports = function(options, allowNull = false) {
 
     if (aliasCount === 0) {
       throw new FirebaseError(
-        "No project active. Run with " +
-          clc.bold("--project <projectId>") +
-          " or define an alias by\nrunning " +
-          clc.bold("firebase use --add"),
+        "No currently active project.\n" +
```

Comment:
You know what would be nice? If you could provide an option to FirebaseError that included additional help text that could be printed after the error message. But that's further down the road...

Reply:
Good idea - I might implement this the next time I have a bug that would make use of it, and then slowly switch over the places in our existing code that could use it.